app/controllers/posts_controller.rb

```diff
@@ -1,17 +1,15 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    # @posts = @user.posts
   end
```

Comment:
- [ ] I would recommend solving the `N+1` problem when fetching all posts and their comments for a user. In the Post's index page, you should use `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`. I can see you have user `@user.posts` directly in the view.